ansible/roles/satellite-configuration/tasks/03_content_view.yml

```diff
@@ -0,0 +1,85 @@
+---
+- name: Fetching list of names available in Content-Views
+  shell: hammer --output cve content-view list --organization "{{ORG}}" |grep Name
+  register: content_view_list
+  ignore_errors: yes
+
+- name: Creating list fact of names available in Content-Views
+  set_fact: 
+    content_view_exist: "{{ content_view_exist + [item.split(':')[1].lstrip(' ')]}}"
+  loop: "{{content_view_list.stdout_lines}}"
```

Comment:
What happen to this tasks and the following if `Fetching list of names available in Content-Views` fails ?
(asking because `ignore_errors` is set to true to it)

Reply:
Fixed